app/uk/gov/hmrc/lisaapi/models/AnnualReturn.scala

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.lisaapi.models
+
+import org.joda.time.DateTime
+import play.api.libs.functional.syntax._
+import play.api.libs.json.{JsPath, Json, Reads, Writes}
+import uk.gov.hmrc.lisaapi.LisaConstants
+import uk.gov.hmrc.lisaapi.controllers.ErrorValidation
+import uk.gov.hmrc.lisaapi.services.CurrentDateService
+
+import scala.collection.mutable.ListBuffer
+
+case class AnnualReturnSupersede (
+  originalLifeEventId: LifeEventId,
+  originalEventDate: DateTime
+)
+
+case class AnnualReturn (
+  eventDate: DateTime,
+  isaManagerName: IsaManagerName,
```

Comment:
I can see it's IsaManagerName in the writes, do we also want it to be IsaManagerName for users (as it is currently) or LisaManagerName?